tachikoma-rest/src/main/kotlin/com/sourceforgery/tachikoma/rest/tracking/TrackingRest.kt

```diff
@@ -126,7 +124,6 @@ internal class TrackingRest(
 
             val notificationMessageBuilder =
                 DeliveryNotificationMessage.newBuilder()
-                    .setCreationTimestamp(emailStatusEvent.dateCreated!!.toTimestamp())
```

Comment:
Not needed anymore?

Reply:
Was always overriden in the send in the sending method for some reason. Kept that logic and removed this, since it's overriden anyways